### ISSUE TITLE
Implement a decaying learning rate

### ISSUE THREAD

**Issue by vluechinger** [tarrade/proj_multilingual_text_classification#55]
- [x] Would it be beneficial to be able to choose between different types of decay (not just exponential)?
- [x] How to be able to see the changing learning rate in the logs?
- [x] Which is preferable: tf.keras.callbacks.LearningRateScheduler or tf.keras.optimizers.schedules.ExponentialDecay? (both should work now but I could not see the learning rate)

**Comment by tarrade** [comment]
One issue to understand but the implementatiuon is done
